bindata/v4.1.0/kube-apiserver/pod.yaml

```diff
@@ -17,6 +17,8 @@ spec:
         - mountPath: /var/log/kube-apiserver
           name: audit-dir
       command: ['/usr/bin/timeout', '105', '/bin/bash', '-ec'] # a bit more than 60s for graceful termination + 35s for minimum-termination-duration, 5s extra cri-o's graceful termination period
+      securityContext:
+        privileged: true
```

Comment:
is this also needed for the bootstrap pot or don't we have init containers there?

Reply:
Also don't forget kube-controller-manager and kube-scheduler.

Reply:
If a container in the pod wants to touch something on the host, then it should be privileged or be given the right label

Reply:
a quick check of kube-controller-manager and kube-scheduler seem fine. Their init containers only wait on ports.

Reply:
I don't know where the bootstrap pod you're wondering about lives

Reply:
https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/bindata/bootkube/bootstrap-manifests/kube-apiserver-pod.yaml

No init container.